tests/functional_tests/test_data_prep.py

```diff
@@ -0,0 +1,9 @@
+import pytest
+
+from src.data_prep import income_cat_proportions
```

Comment:
imports should be from package name
Example if package name is housepred do from housepred.data_prep import xyz

Reply:
done